### ISSUE TITLE
Error Message Popup in Windows 7 32-bit

### ISSUE THREAD

**Issue by davidpgil** [canton7/SyncTrayzor#602]
```
Version: 1.1.26.0; Variant: Portable; Arch: X86
Path: C:\Users\davidphilipegil\scoop\apps\synctrayzor\current\SyncTrayzor.exe
System.BadImageFormatException: Could not load file or assembly 'CefSharp.Core.dll' or one of its dependencies.  is not a valid Win32 application. (Exception from HRESULT: 0x800700C1)
File name: 'CefSharp.Core.dll'
   at SyncTrayzor.Pages.ViewerViewModel.OnInitialActivate()
   at Stylet.Screen.<Stylet.IScreenState.Activate>b__35_0(ScreenState oldState, ScreenState newState)
   at Stylet.Screen.SetState(ScreenState newState, Action`2 changedHandler)
   at Stylet.Screen.Stylet.IScreenState.Activate()
   at Stylet.ScreenExtensions.<>c__DisplayClass4_0.<ActivateWith>b__0(Object o, ActivationEventArgs e)
   at System.EventHandler`1.Invoke(Object sender, TEventArgs e)
   at Stylet.Screen.<>c__DisplayClass35_0.<Stylet.IScreenState.Activate>b__1()
   at Stylet.Execute.OnUIThread(Action action)
   at Stylet.Screen.<Stylet.IScreenState.Activate>b__35_0(ScreenState oldState, ScreenState newState)
   at Stylet.Screen.SetState(ScreenState newState, Action`2 changedHandler)
   at Stylet.Screen.Stylet.IScreenState.Activate()
   at Stylet.ScreenExtensions.TryActivate(Object screen)
   at Stylet.WindowManager.WindowConductor..ctor(Window window, Object viewModel)
   at Stylet.WindowManager.CreateWindow(Object viewModel, Boolean isDialog, IViewAware ownerViewModel)
   at Stylet.WindowManager.ShowWindow(Object viewModel)
   at SyncTrayzor.Pages.ShellViewModel.EnsureInForeground() in C:\projects\synctrayzor\src\SyncTrayzor\Pages\ShellViewModel.cs:line 184
   at SyncTrayzor.Services.ApplicationWindowState.EnsureInForeground() in C:\projects\synctrayzor\src\SyncTrayzor\Services\ApplicationWindowState.cs:line 62
   at SyncTrayzor.NotifyIcon.NotifyIconManager.<.ctor>b__37_1(Object o, EventArgs e) in C:\projects\synctrayzor\src\SyncTrayzor\NotifyIcon\NotifyIconManager.cs:line 107
   at SyncTrayzor.NotifyIcon.NotifyIconViewModel.OnWindowOpenRequested() in C:\projects\synctrayzor\src\SyncTrayzor\NotifyIcon\NotifyIconViewModel.cs:line 178
   at SyncTrayzor.NotifyIcon.NotifyIconViewModel.Restore() in C:\projects\synctrayzor\src\SyncTrayzor\NotifyIcon\NotifyIconViewModel.cs:line 141
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at Stylet.Xaml.ActionBase.InvokeTargetMethod(Object[] parameters)
   at Stylet.Xaml.CommandAction.Execute(Object parameter)
   at MS.Internal.Commands.CommandHelpers.CriticalExecuteCommandSource(ICommandSource commandSource, Boolean userInitiated)
   at System.Windows.Controls.MenuItem.InvokeClickAfterRender(Object arg)
   at System.Windows.Threading.ExceptionWrapper.InternalRealCall(Delegate callback, Object args, Int32 numArgs)
   at System.Windows.Threading.ExceptionWrapper.TryCatchWhen(Object source, Delegate callback, Object args, Int32 numArgs, Delegate catchHandler)
```

**Comment by DLasher106** [comment]
Same thing on Windows 10 32-bit.

**Comment by canton7** [comment]
@DLasher106 Do you use the portable version as well?

**Comment by canton7** [comment]
Can you try this version, please? https://ci.appveyor.com/project/canton7/synctrayzor/builds/37068269/artifacts . Thanks!

**Comment by skylineKid712** [comment]
@canton7 that version works fine for me. Not crashing on startup anymore

**Comment by canton7** [comment]
Thanks for confirming!

**Comment by canton7** [comment]
I'm going to wait for a couple more confirmations before releasing that version: the previous release fixed this issue for some people, but apparently not others!

**Comment by davidpgil** [comment]
Seems to work for me as well. I dont't know if you are the ones to add it to the scoop extras repo, but if you are, please update there as well.

**Comment by canton7** [comment]
Thanks for confirming!

I'd never heard of scoop: thanks for the heads-up. The chocolatey package is official, but the scoop one isn't. They picked up the 1.1.26 release quickly though

**Comment by DLasher106** [comment]
1.1.26.584 is working here, too.
(Setup version, in Windows 10 x32 running in VMWare Player.)

**Comment by canton7** [comment]
Fixed in 1.1.27
